interactive/migrations/0021_copy_organisations_to_memberships.py

```diff
@@ -0,0 +1,42 @@
+# Generated by Django 4.1.4 on 2022-12-16 08:54
+from django.db import migrations
+from django.utils.text import slugify
+
+
+def add_orgs_and_memberships(apps, schema_editor):
+    Org = apps.get_model("interactive", "Org")
+    OrgMembership = apps.get_model("interactive", "OrgMembership")
+    User = apps.get_model("interactive", "User")
+
+    users_with_orgs = User.objects.exclude(organisation="")
+
+    # create the orgs
+    org_names = set(users_with_orgs.values_list("organisation", flat=True))
+    Org.objects.bulk_create([Org(name=name, slug=slugify(name)) for name in org_names])
```

Comment:
I don't think this is a good idea and that we should do it manually. I think we should delete all disabled user accounts very soon, which also only leaves Bennett staff anyway. Also, the organisation field was free text, so we'll end up with a lot of duplicated/nonsensical organisations

Reply:
Good point, I'll drop this migration 👍